easybuild/easyconfigs/l/libgd/libgd-2.3.3-GCCcore-11.2.0.eb

```
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'libgd'
+version = '2.3.3'
+
+homepage = 'https://libgd.github.io'
+description = "GD is an open source code library for the dynamic creation of images by programmers."
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/libgd/libgd/releases/download/gd-%(version)s/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['dd3f1f0bb016edcc0b2d082e8229c822ad1d02223511997c80461481759b1ed2']
+
+builddependencies = [
+    ('binutils', '2.37'),
+]
+
+dependencies = [
+    ('fontconfig', '2.13.94'),
+    ('libjpeg-turbo', '2.0.6'),
+    ('libpng', '1.6.37'),
+    ('zlib', '1.2.11'),
+]
+
+configopts = "--with-fontconfig=$EBROOTFONTCONFIG --with-jpeg=$EBROOTLIBJPEGMINTURBO "
+configopts += "--with-png=$EBROOTLIBPNG --with-zlib=$EBROOTZLIB"
+
+sanity_check_paths = {
+    'files': ['lib/libgd.a', 'lib/libgd.%s' % SHLIB_EXT],
+    'dirs': ['bin', 'include'],
+}
+
```

@boegel most libgd tools don't seem to support `--help` or `--version`, but `webpng` does:
```
$ webpng --help
Usage: webpng [-i y|n] [-l] [-t index|none] [-d] [-a] pngname.png
  -i <y|n>   Turns on/off interlace
  -l         Prints the table of color indexes
  -t <index> Set the transparent color to the specified index (0-255 or "none")
  -d         Reports the dimensions and other characteristics of the image
  -a         Prints all alpha channels that are not 100% opaque

If you specify '-' as the input file, stdin/stdout will be used as input/output.
$ echo $?
0
```

Should we add that one?
```suggestion

sanity_check_commands = ['webpng --help']

```